Integrations/integration-Fortigate.yml

```diff
@@ -0,0 +1,1381 @@
+description: Manage FortiGate Firewall
+configuration:
+- display: Server URL (e.g. 192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Account username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Trust any certificate (unsecure)
+  name: unsecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+
+    import json
+    import requests
+
+    # Disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    ''' GLOBALS/PARAMS '''
+
+    USER_NAME = demisto.params().get('credentials').get('identifier')
+    PASSWORD = demisto.params().get('credentials').get('password')
+    SERVER = demisto.params()['server'][:-1] if demisto.params()['server'].endswith('/') else demisto.params()['server']
+    USE_SSL = not demisto.params().get('unsecure', False)
+    BASE_URL = SERVER + '/api/v2/'
+
+    # remove proxy if not set to true in params
+    if not demisto.params().get('proxy'):
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+
+    ''' HELPER FUNCTIONS '''
+
+
+    def get_request(url_suffix, params={}):
+        """
+        Used in order to perform get operations - retrieve data.
+        """
+        session = login()
+        try:
+            res = session.get(
+                BASE_URL + url_suffix,
+                verify=USE_SSL,
+                params=params
+            )
+            logout(session)
```

Comment:
Please move this line `logout(session)` to be in the `finally` section in the `try-except` of the commands